src/main/java/org/tron/common/runtime/vm/PrecompiledContracts.java

```diff
@@ -90,6 +108,12 @@ public static PrecompiledContract getContractForAddress(DataWord address) {
         public abstract long getGasForData(byte[] data);
 
         public abstract Pair<Boolean, byte[]> execute(byte[] data);
+
+        public static byte[] callerAddress;
+
+        public static Deposit deposit;
+
+        public static ProgramResult result;
```

Comment:
![Vulnerability](https://sonarsource.github.io/sonar-github/vulnerability.png 'Vulnerability') Vulnerability: Make result a static final constant or non-public and provide accessors if needed. ([squid:ClassVariableVisibilityCheck](https://sonarcloud.io/organizations/tron-zhaohong/rules#rule_key=squid%3AClassVariableVisibilityCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=java-tron&pullRequest=1172&issues=AWTGhtDs6dA5b-W23TXO&open=AWTGhtDs6dA5b-W23TXO)

[ISSUE_KEY]: AWTGhtDs6dA5b-W23TXO